src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.styled.tsx

```diff
@@ -1,18 +1,5 @@
 import styled from 'styled-components';
 
-const Container = styled.div`
-  width: 100%;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-
-  @media screen and (max-width: ${({ theme }) => theme.widths.tablet}) {
-    & {
-      order: 2;
-    }
-  }
-`;
-
 const ShortcutsWrapper = styled.div`
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏When you added this `ShortcutWrapper` to `CitiesWeatherWidget`, shouldn't you have removed it here?

Reply:
That's true, indeed :) 
Thanks :)

Reply:
Done 👍